bento_wes/backends/wes_backend.py

```diff
@@ -46,25 +46,28 @@ def __init__(
         tmp_dir: Path,
         data_dir: Path,
         workflow_timeout: int,  # Workflow timeout, in seconds
+        settings: Settings,
```

Comment:
a lot of the other arguments in this constructor come directly from Settings. we should either just accept a Settings object or put everything we pull from Settings as a separate argument, up to you which.